pkg/trie/db/db.go

```diff
@@ -22,6 +22,11 @@ type NewBatcher interface {
 	NewBatch() database.Batch
 }
 
+type RWDatabase interface {
```

Comment:
Since Database if the getter plus the putter, and the writer used hear already contains the put method, and we just remove this interface and change `Database` to be `type RWDatabase interface {
	DBGetter
	Writer
}` and just use that everywhere instead?

Reply:
The difference is that `RWDatabase` also includes the `Del`